test-data/unit/check-isinstance.test

```diff
@@ -423,8 +565,6 @@ def foo() -> None:
     x = 1 # type: Union[int, str]
     if isinstance(x, int):
         return
-    else:
-        pass
```

Comment:
Curious about this net change, was it intentional? It would be good to have tests for both the with-`else` and without-`else` versions (but maybe one is covered by another test already).